cpt/runner.py

```diff
@@ -214,12 +216,13 @@ def run(self, pull_image=True, docker_entry_script=None):
         else:
             update_command = ""
 
-        command = ('%s docker run --rm -v "%s:%s/project" %s %s %s %s '
+        command = ('%s docker run --rm -v "%s:%s/project" %s %s %s %s %s '
```

Comment:
Please, update the tests to validate your change.

Reply:
Done with the exact test from readme example, checking that the freshly created directory is empty and after docker run contains the directory for a created package.